src/System.Windows.Extensions/tests/System/Media/SoundPlayerTests.cs

```diff
@@ -80,7 +80,7 @@ public static IEnumerable<object[]> Play_InvalidString_TestData()
             yield return new object[] { "ima.wav" };
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(Helpers.IsSoundPlaySupported))]
```

Comment:
This condition will fail, you're saying that the property Helpers.IsSoundPlaySupported is defined in PlatformDetection type. Just remove Helpers at all in this case.